config/foreman-proxy.yaml

```diff
@@ -0,0 +1,15 @@
+---
+:name: Foreman Proxy
+:description: Install a stand-alone Foreman Proxy.
+:log_dir: './_build/'
+:log_name: foreman-proxy.log
+:log_level: :debug
+:answer_file: ./config/foreman-proxy-answers.yaml
+:installer_dir: .
+:module_dirs: ./_build/modules
+:parser_cache_path: ./_build/parser_cache/foreman.yaml
```

Comment:
This should be `foreman-proxy`

Reply:
`foreman-proxy-content` doesn't have its own one. I thought this would just use `foreman.yml` ?

https://github.com/Katello/katello-installer/blob/master/config/foreman-proxy-content.yaml#L14-L16

Reply:
As long as the set of modules in the answer file is a subset of those in the foreman scenario this can be re-used like this.

Reply:
I'm OK with using `foreman.yml` given I don't expect this to be stand-alone. @ekohl thoughts?

Reply:
I must admit I don't know enough about the exact parser cache. In the Rakefile I built it so it generates a cache file per scenario. If you add it, then it should generate the right configs and fix the config file in the build process: https://github.com/theforeman/foreman-installer/blob/7eb7ca11fecb88d7f785e504903db4e3350fc03a/Rakefile#L59